src/app/components/form/form.component.ts

```diff
@@ -41,7 +46,21 @@ export class FormComponent implements OnInit {
             keywords: sleutelwoorden
         };
 
-        this.http.post(environment.api + '/searchrequest', body).subscribe((data: any) => alert(data.queries));
-    };
+        // this.http.post(environment.api + '/searchrequest', body).subscribe((data: any) => {
+        this.http.get(environment.api + '/getAllJobs').subscribe((data: any) => {
+            console.log(data);
+            data.forEach(val => {
+               this.ELEMENT_DATA.push({
+                   title: val.title,
+                   broker: val.broker,
+                   postingDate: val.postingDate,
+                   location: val.location,
+                   url: val.id
+               });
```

Comment:
Could be a bit clearer this way:
```suggestion
            data.forEach(vacancy => {
               this.ELEMENT_DATA.push({
                   title: vacancy.title,
                   broker: vacancy.broker,
                   postingDate: vacancy.postingDate,
                   location: vacancy.location,
                   url: vacancy.id
               });
```